app/controllers/concerns/fraud_review_concern.rb

```diff
@@ -5,12 +5,25 @@ def handle_pending_fraud_review
     redirect_to_fraud_review if fraud_review_pending?
   end
 
+  def handle_fraud_rejection
```

Comment:
Maybe this should be automatically wired up as a `before_action` if the concern is included:

```ruby
included do
  before_action :handle_fraud_rejection
end
```

Taking a quick look over where this concern is included, I don't see a downside? We're having a proliferation of controllers under `Idv` and it'd be nice for them all to be able to easily redirect users due to being rejected.

Reply:
Idv::SetupErrorsController should get this `before_action` as well--if you refresh the page after being rejected, it should catch that and redirect you to the new messaging.

Reply:
true. yeah that will fix up some DRY.